### PR TITLE
combine note, description textedit [CPP-937]

### PR DESCRIPTION
**Pull request by adrian-kong**
image

### CODE REVIEW

File: resources/SettingsTabComponents/SettingsPane.qml [hunk 1]
```diff
@@ -314,7 +277,11 @@ Rectangle {
             anchors.fill: parent
 
             TextEdit {
-                text: selectedRowField(_fieldName)
+                text: {
+                    if (_fieldName == "description")
+                        return selectedRowField(_fieldName) + "\n\nNotes:\n" + selectedRowField("notes");
```
Comment (silverjam) [inline resources/SettingsTabComponents/SettingsPane.qml:282]: Can `Notes:` be moved into the `Description:` row label?

image

e.g. `property string _title: "Description"` would become `property string _title: "Description\n\nNotes:"`

Comment (adrian-kong) [reply resources/SettingsTabComponents/SettingsPane.qml:282]: yeah but dont think the notes would always be aligned with the text inside the area tho

File: resources/SettingsTabComponents/SettingsPane.qml [hunk 2]
```diff
@@ -217,60 +217,23 @@ Rectangle {
...
-
-                property string _fieldName: "notes"
-
-                visible: !!selectedRowField(_fieldName)
-                Layout.columnSpan: parent.columns - 1
-                Layout.rowSpan: parent.rows - 8
                 Layout.preferredHeight: Math.max(1, parent.height - 8 * parent.smallRowHeight)
-                Layout.preferredWidth: parent.colWidthField
                 sourceComponent: settingRowText
             }
```
Comment (adrian-kong) [inline resources/SettingsTabComponents/SettingsPane.qml:236]:
```suggestion
            }
            
```

File: resources/SettingsTabComponents/SettingsPane.qml [hunk 3]
```diff
@@ -314,7 +277,11 @@ Rectangle {
             anchors.fill: parent
 
             TextEdit {
-                text: selectedRowField(_fieldName)
+                text: {
+                    if (_fieldName == "description")
+                        return selectedRowField(_fieldName) + "\n\nNotes:\n" + selectedRowField("notes");
+                    return selectedRowField(_fieldName)
```
Comment (adrian-kong) [inline resources/SettingsTabComponents/SettingsPane.qml:283]:
```suggestion
                        return selectedRowField(_fieldName) + "\n\nNotes:\n" + selectedRowField("notes");
                        
                    return selectedRowField(_fieldName)
```

**Comment by silverjam** [comment]
@switanis Does this look OK?

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/5009452937)

**Comment by silverjam** [comment]
@adrian-kong Let's publish a test release for @switanis to try out.

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/5009853917)

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/5019999175)

**Review by silverjam — APPROVED** [review]

**Comment by silverjam** [comment]
@adrian-kong need to forward port this change (to `main` branch)

**Pull request merged**